openvino/conditional_compilation/CMakeLists.txt

```diff
@@ -36,8 +36,16 @@ if(SELECTIVE_BUILD STREQUAL "COLLECT")
     FetchContent_GetProperties(ext_seapi)
     if(NOT ext_seapi_POPULATED)
         FetchContent_Populate(ext_seapi)
+        if (X86_64)
+            set (SEAPI_FORCE_BITS 64)
+        elseif(X86)
+            set (SEAPI_FORCE_BITS 32)
```

Comment:
`if(CMAKE_SIZEOF_VOID_P EQUAL 8)` ?

Reply:
I believe this is already done in one target_flags.cmake plus it covers cross build.